What’s up in Vietnam.md

```diff
@@ -1,62 +1,63 @@
+Follow the definition on Wikipedia _it’s a technique for converting data between incompatible type systems using object-oriented programming languages._ To be more precise it’s converting relational data to the object-oriented domain model (OODM) and OODM to relational data. Conversions like that can be done in a several ways, but the most common approach is to use a tool called ORM library or ORM framework.
 
-## Why is it problematic.
-There are words such as „incompatible type systems” even in the definition, and all incompatibilities are a potential problem. My first question which appeared after getting into the subject was „Why the hell we are using incompatible systems? Why is it the first (and often the only one known) solution to solve the problem with persisting data?”
+## Why is it problematic?
+ „Incompatible type systems” are mentioned already in the definition, and all incompatibilities are potential problems. My first question that popped into my head after getting into the subject was: „Why the hell we are using incompatible systems? Why is it the first (and often the only one known) solution to solve the problem with persisting data?”
 
-> If your project really does not need any relational data features, then ORM will work perfectly for you, but then you have a different problem: you're using the wrong data store. […] On the other hand, if your data is relational, then your object mapping will eventually break down.  
+> If your project really does not need any relational data features, then ORM will work perfectly for you, but then you have a different problem: you're using the wrong data store. […] On the other hand, if your data is relational, then your object mapping will eventually break down.
 _Laurie Voss_
 
 ### Let’s characterize these two worlds
 
-> Object systems are typically characterized by four basic components: identity, state, behavior and encapsulation.  
+> Object systems are typically characterized by four basic components: identity, state, behavior and encapsulation.
 _Ted Neward_
 
-Summing up the above thought, we can say that every object is described by its identity independently of its state. Every object has a collection of operations that can be invoked to manipulate the state of itself or connected objects, and (most importantly) has encapsulated access to itself and effectively, the ability to inheritance and polymorphism.
+Elaborating on the above though we can say that every object is described by its identity independently of its state. Every object has a collection of operations that can be invoked to manipulate the state of itself or connected objects, and (most importantly) has encapsulated access to itself and effectively, possibility of inheritance and polymorphism.

```

Comment:
I still think that it's ability of an object to inheritance and polymorphism, not a possibility. Can't it stay like it way?

Reply:
then it needs to say "ability to inherit and polymorph"

Reply:
I've changed it.